apps/src/p5lab/P5Lab.js

```diff
@@ -732,6 +732,11 @@ P5Lab.prototype.startTickTimer = function() {
  *     implementation.
  */
 P5Lab.prototype.resetHandler = function(ignore) {
+  if (!this.isSpritelab) {
```

Comment:
I would rather see these changes go in GameLab.js if possible. Can you override these methods there without breaking things?